DATA/production/qc-async/fv0.json

```
@@ -21,26 +21,48 @@
         "conditionDB": {
           "url": "qcdb.cern.ch:8083"
         }
-      },
-      "tasks": {
-        "FV0Digits": {
-          "active": "true",
-          "taskName": "Digits",
-          "className": "o2::quality_control_modules::fv0::DigitQcTask",
-          "moduleName": "QcFV0",
-          "detectorName": "FV0",
-          "cycleDurationSeconds": "60",
-          "maxNumberCycles": "-1",
-          "dataSource": {
-            "type": "direct",
-            "query": "digits:FV0/DIGITSBC/0;channels:FV0/DIGITSCH/0"
-            },
-          "taskParameters": {
-            "ChannelIDs": "0,1,2,3,4,5,6,7,8,9,10,11"
-          }
+    },
+    "tasks": {
+      "DigitQcTask": {
+        "active": "true",
```

```suggestion
      "FV0Digits": {
        "active": "true",
        "taskName": "Digits",
```
Could you please keep FV0Digits and "taskName": "Digits" as they were? Otherwise, do you have a reason to break the recently established naming convention?